containers/test-apps/Dockerfile

```diff
@@ -3,14 +3,11 @@ FROM twosigma/mesos-agent:1.0.0-1.0.1604
 # we need nginx for our http/2 tests
 # gettext-base is needed for envsubst in the nginx server
 # python is needed to launch kitchen
-RUN apt-get update \
-  && apt-get install -y gettext-base \
-  && apt-get install -y nginx \
-  && apt-get install -y python3
+RUN apt-get update && apt-get install -y gettext-base nginx python3
 
-COPY kitchen/bin/waiter-init /usr/bin/waiter-init
-COPY kitchen/bin/kitchen /opt/kitchen/kitchen
+COPY waiter-init/bin/waiter-init /usr/bin/waiter-init
 COPY nginx/bin/run-nginx-server.sh /opt/nginx/bin/run-nginx-server.sh
 COPY nginx/data/nginx-template.conf /opt/nginx/data/nginx-template.conf
+COPY kitchen/bin/kitchen /opt/kitchen/kitchen
```

Comment:
Nitpick: why was this moved lower down than in the previous version?

Reply:
I think this is an artifact of me splitting this off from my WAITER_SANDBOX PR. I'll sort these lexicographically instead.